pipelines/rj_ssm__compstat/task.py

```
@@ -0,0 +1,99 @@
+
+    Returns:
+        json: Dados extraídos da chave especificada no parâmetro 'result'.
+    """
+
+    url = f"{url_base}{endpoint}"
+
+    params = {
+        "token" : token_estatic
+    }
+
+    response = requests.get(url, params=params)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)